src/app/components/select-field/index.html

```diff
@@ -166,31 +166,32 @@
       propertyName="data"
       isOptional="true"
     >
-      An array of <a stacheRouterLink="." fragment="skyselectfield-properties">The <sky-code>SkySelectField</sky-code> objects</a> provided to the select field's <sky-code>data</sky-code> property. Use this property to populate your custom picker with options for the user to select.
+      Receives an array <a stacheRouterLink="." fragment="skyselectfield-properties">of <sky-code>SkySelectField</sky-code> objects</a> from the select field's <sky-code>data</sky-code> property. Use this property to populate the custom picker with items for users to select.
     </sky-demo-page-property>
     <sky-demo-page-property
       propertyName="headingText"
       isOptional="true"
     >
-      The text provided to the select field's <sky-code>headingText</sky-code> property. If your custom picker is a modal, use this property for the modal's title.
+      Receives text from the select field's <sky-code>pickerHeading</sky-code> property. Use this property to display a header in the custom picker's modal.
```

Comment:
Not all custom pickers must be modal, so I think it's still worth mentioning the "if your custom picker is a modal" part.